### PR TITLE
Added checks for session handling

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

**Review by turtledreams — APPROVED** [review]
we can review the changelog before release if needed

File: CountlyConnectionManager.m [hunk 1]
```diff
@@ -11,6 +11,7 @@ @interface CountlyConnectionManager ()
     NSTimeInterval unsentSessionLength;
     NSTimeInterval lastSessionStartTime;
     BOOL isCrashing;
+    BOOL isSessionStarted;
```
Comment (arifBurakDemiray) [inline CountlyConnectionManager.m:14]: does not it require default value as NO

**Review by arifBurakDemiray — APPROVED** [review]
Looks cool

File: CountlyConnectionManager.m (hunk 1 above)
Comment (ijunaid) [reply CountlyConnectionManager.m:14]: > does not it require default value as NO

Good catch, not sure, but i have set default value to NO in init

**Pull request merged**